### ISSUE TITLE
Improve deployment documentation

### ISSUE THREAD

**Issue by Tansito** [Qiskit-Extensions/quantum-serverless#1274]
### What is the expected enhancement?

From last interaction with Johannes and Paco, they asked to update the documentation to deploy in IBM Cloud. Basically we followed the next steps:

- IBM Cloud setup
  - Make sure to be part of an Access Group with admin rights
  - Create Kubernetes Service with minimal resources
    - Create VPC
    - 4 CPU cores minimum
  - Create Object Storage
  - Create minimal PostgreSQL instance
    - 0 dedicated cores and 1 GB Ram is fine for testing purposes
- Open IBM Cloud Shell
  - Set Context: ibmcloud ks cluster config --cluster $YOUR_CLUSTER_ID
  - Get cluster ID from the overview of the Kubernetes Service resource you created
  - Create namespace: kubectl create namespace serverless-namespace
- Next step is configure object storage and database to be able to access to them
  - for PostgreSQL you need: click in the database > service credentials > new credential
  - for Object Storage you need:
    - create credentials: enter in the instance > service credentials > new credentials with: role writer and HMAC ON
    - create bucket: custom bucket > default configuration is more than enough
 - Steps to configure object storage plugin in the cluster:
   - create the secret: https://cloud.ibm.com/docs/containers?topic=containers-storage-cos-understand#create_cos_secret
   - install the plug-in: https://cloud.ibm.com/docs/containers?topic=containers-storage_cos_install
 - Install Quantum Serverless

**Comment by johannesgreiner** [comment]
yes this is great thx David for the joint effort in updating! Also whatever insights we can add on deploying in other Cloud contexts is helpful, I'll scan through my materials on this as well
